client/src/components/_common/UserSearchbar/UserSearchbar.js

```diff
@@ -11,7 +12,8 @@ const UserSearchbar = ({
   addDisabled,
   searchDisabled,
   searchResults,
-  placeholder
+  placeholder,
+  allocationManagerLoading
```

Comment:
Seeing as this is a common component, we should rename this to something more generic, like `onAddLoading` or just `loading`